config/rbac/lighthouse-agent/ocp_cluster_role.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
```

Comment:
All the clusterRoles are exactly the same except for the name. So, can we create a single `ClusterRole` and use separate `ClusterRoleBindings` to assign the clusterRole to various components?

CC: @skitt

Reply:
I created a separate clusterrole per component coz the plan is to move all the ocp relevant permissions to the relevant ClusterRole per component, for example [gateway ocp permissions](https://github.com/submariner-io/submariner-operator/blob/devel/config/rbac/submariner-gateway/cluster_role.yaml#L37-#L52) should move to gateway clusterrole file.

We'll have a follow-up PR to address that.

Having a separate file per component allows flexibility in the future.

Reply:
Maybe we should have a single RBAC file per component for both k8s and ocp and generate the OCP part conditionally on subctl side before applying it. 

Not sure if we can deliver it for 0.14 (maybe we can file an enhancement) , we need to unblock submariner deployments on ROSA and ARO as soon as possible to start testing.

Reply:
I think it’s fine to have one file per component, matching the other `ClusterRole` definitions. As Yossi says above, they won’t all stay the same anyway.